schedule/yast/allmodules+allpatterns+registration/allmodules+allpatterns+registration.yaml

```diff
@@ -1,9 +1,18 @@
 ---
-name:           skip_registration
+name:           allmodules+allpatterns+registration
 description:    >
-  Like a standard scenario with explicit skipping of SCC registration
-  in case where we register by default, e.g. for SLE >= 15
-  See https://progress.opensuse.org/issues/25264 for details.
+  Full Medium installation that covers the following cases:
+     1. Additional modules enabled using SCC (Legacy, Development Tools, Web and
+        Scripting, Containers, Desktop Applications);
```

Comment:
Where are the additional modules enabled in your VRs? https://openqa.suse.de/tests/3979964#step/addon_products_sle/28. For example in functional group: https://openqa.suse.de/tests/3983204#step/scc_registration/27 they are using `ENABLE_ALL_SCC_MODULES`

Reply:
Nice catch, I thought `ADDONS: all-packages` doing that. Changed the variable and restarted VRs. I'll place the PR to WIP until the VRs will be finished.

Reply:
Done.